test/protocol.spec.ts

```diff
@@ -40,7 +40,24 @@ describe('protocol-contract', async () => {
     });
   });
 
-  it('does not allow protocol upgrade to be scheduled before the latest upgrade schedule', async () => {
+  it('does not allow protocol upgrade to be scheduled before the latest upgrade schedule when latest upgrade already took place', async () => {
```

Comment:
isnt this one covered completely by `it('does not allow protocol upgrade to be scheduled in the past'`?

Reply:
if you leave both then just express the difference clearly in the description..?